psydac/feec/multipatch/examples/mixed_source_pbms_conga_2d.py

```diff
@@ -413,8 +413,9 @@ def P2_phys(f_phys):
     # bc_type = 'pseudo-vacuum'
     source_type = 'dipole_J'
 
-    source_proj = 'P_L2_wcurl_J'
-
+    # source_proj = 'P_L2_wcurl_J'
+    source_proj = 'P_geom'
+    
```

Comment:
What example do you want to be run manually? Is this the one presented in the paper? If yes, we can maybe add a comment here.

Reply:
both should be fine, I just put the fastest I guess